sdk/src/Services/SimpleNotificationService/Custom/AmazonSimpleNotificationServiceClient.Extensions.cs

```diff
@@ -39,7 +32,7 @@ private static void AddSQSPermission(Policy policy, string topicArn, string sqsQ
         {
             Statement statement = new Statement(Statement.StatementEffect.Allow);
 #pragma warning disable CS0612,CS0618
```

Comment:
The `pragma warning disable CS0612,CS0618` can be removed here too, right?

---

Not related to this PR, but there are also a lot of places with these `pragma` for `AWSSDKUtils.CorrectedUtcNow` (which we unmarked as deprecated); it'd be nice to remove them for `v4` as well.